src/box/lua/console.c

```diff
@@ -226,13 +227,29 @@ console_push_line(char *line)
 	free(line);
 }
 
+static bool sigint_called = false;
+static struct fiber *interactive_fb;
+
+static void
+console_sigint_handler(ev_loop *loop, struct ev_signal *w, int revents)
```

Comment:
Please add a comment explaining why we need it.

Reply:
Added.